Sources/AccountSDKIOSWeb/Simplified Login/Sub Views/FooterView.swift

```diff
@@ -116,3 +118,19 @@ class FooterView: UIStackView {
         return view
     }
 }
+
+private extension UIStackView {
+  func reverseSubviewsZIndex(setNeedsLayout: Bool = true) {
+    let stackedViews = self.arrangedSubviews
+    stackedViews.forEach {
+      self.removeArrangedSubview($0)
+      $0.removeFromSuperview()
+    }
+    stackedViews.reversed().forEach(addSubview(_:))
+    stackedViews.forEach(addArrangedSubview(_:))
+
+    if setNeedsLayout {
+      stackedViews.forEach { $0.setNeedsLayout() }
+    }
+  }
```

Comment:
Nice. Could be indented
```suggestion
private extension UIStackView {
    func reverseSubviewsZIndex(setNeedsLayout: Bool = true) {
        let stackedViews = self.arrangedSubviews
        stackedViews.forEach {
            self.removeArrangedSubview($0)
            $0.removeFromSuperview()
        }
        stackedViews.reversed().forEach(addSubview(_:))
        stackedViews.forEach(addArrangedSubview(_:))
        
        if setNeedsLayout {
            stackedViews.forEach { $0.setNeedsLayout() }
        }
    }
}
```

Reply:
done